NugetUpdate/Repositories/AzureDevOps.cs

```diff
@@ -16,14 +17,18 @@ public class AzureDevOps : IRepositorySource, IDisposable
         readonly string _apiBase;
         private readonly string _defaultBranch;
         readonly TextWriter _log;
+        private readonly string _areaPath;
+        private readonly IWorkItemService _workItemService;
 
-        public AzureDevOps(string token, string organization, string project, string repo, string defaultBranch, TextWriter log)
+        public AzureDevOps(string token, string organization, string project, string repo, string defaultBranch, TextWriter log, string areaPath = "Health\\Healthcare APIs\\Meru")
```

Comment:
Perhaps leave null as default here.

```suggestion
        public AzureDevOps(string token, string organization, string project, string repo, string defaultBranch, TextWriter log, string areaPath = null)
```